### DISCUSSION TITLE
Option when purchasing item, to auto hold crates with purchased item, at start of next round,

### DISCUSSION THREAD

**Discussion by Asadgunz** [FakeFishGames/Barotrauma#10808] [Ideas and suggestions]
Add a simple option, when paying for it at vendor, like "For me" or something like that, so that when the round start, you'll be holding the crate with said items. Useful when medics purchase their own medical supplies, to start with it, instead of getting rob by loot goblins in the sub.

**Comment by HSCyber** [comment]
您好，您发给我的邮件已收到，谢谢。
